src/remoted/main.c

```diff
@@ -127,13 +125,20 @@ int main(int argc, char **argv)
         merror_exit(CONFIG_ERROR, cfg);
     }
 
+    /* Setting default values and reading global configuration */
+    Config.agents_disconnection_time = 20;
+    Config.alert_agent_disconnection_time = 2;
+    if (ReadConfig(CGLOBAL, cfg, &Config, NULL) < 0) {
```

Comment:
This should be part of RemotedConfig() method, including default values.

Reply:
Done!